packages/plugin-core/src/features/BacklinksTreeDataProvider.ts

```diff
@@ -18,6 +20,129 @@ class Backlink extends vscode.TreeItem {
   }
 }
 
+/**
+ * Given the fsPath of current note, return the list of backlink sources as tree view items.
+ * @param fsPath fsPath of current note
+ * @returns list of the source of the backlinks as TreeItems
+ */
+const pathsToBacklinkSourceTreeItems = async (
+  fsPath: string,
+  isUnrefEnabled: boolean | undefined
+) => {
+  const refFromFilename = path.parse(fsPath).name;
+  const referencesByPath = _.groupBy(
+    await findReferences(refFromFilename, [fsPath]),
+    ({ location }) => location.uri.fsPath
+  );
+
+  const pathsSorted = sortPaths(Object.keys(referencesByPath), {
+    shallowFirst: true,
+  });
+
+  if (!pathsSorted.length) {
+    return [];
+  }
+
+  const collapsibleState = true
+    ? vscode.TreeItemCollapsibleState.Collapsed
+    : vscode.TreeItemCollapsibleState.Expanded;
+
+  return pathsSorted.map((pathParam) => {
```

Comment:
when backlinks aren't enabled, we'll show links(0) in the backlinks panel if only unreferenced links exist. we should show no result instead

Reply:
I just noticed this too while using the panel. I'll fix this.